src/services/platform.service.js

```diff
@@ -44,10 +44,21 @@ export const PACKAGE_MANAGER_CMD = path.join(
 // Forward the host env, and append the
 // project's .bin directory to PATH to allow
 // package scripts to function properly.
-export const getBaseProjectEnvironment = (projectPath: string) => ({
-  ...window.process.env,
+/**
+ * For running tasks in a cross-platform manner, this helper does a few things:
+ *  - Forward the host environment
+ *  - Append the project's .bin directory to PATH to allow package scripts to
+ *    function properly.
+ *  - Add `FORCE_COLOR: true`, so that terminal output includes color codes.
+ */
+export const getBaseProjectEnvironment = (
+  projectPath: string,
+  currentEnvironment: Object = window.process.env
```

Comment:
Added this for testing, since `window.process` is undefined in Jest.

I suppose I could create a mock in Jest for it, actually... maybe I should just do that.

Reply:
Either seems fine to me - the optional argument abstracts the function one step further but I don't think it distracts from the intended purpose at all, and it definitely facilitates testing.